webgl/src/webgl.rs

```diff
@@ -688,4 +688,28 @@ impl GLContext {
             return ctx.texParameterf(@{TextureBindPoint::Texture2d as u32},@{pname as u32},@{param})
         };
     }
+
+    pub fn create_framebuffer(&self)  -> WebGLFrameBuffer {
+        let val = js! {
+            var ctx = Module.gl.get(@{self.reference});
+            return ctx.create_framebuffer();

```

Comment:
If you want to use the fb by Module.gl.get later, you would need to store it in here first. You could use
```
return Module.gl.add(ctx.create_framebuffer()) ; 
```